pilot/pkg/model/destination_rule.go

```diff
@@ -169,13 +169,20 @@ func (l *consolidatedDestRule) Equals(r *consolidatedDestRule) bool {
 	}
 
 	// compare from
-	if len(l.from) != len(r.from) {
+	if len(l.From) != len(r.From) {
 		return false
 	}
-	for i, v := range l.from {
-		if v != r.from[i] {
+	for i, v := range l.From {
+		if v != r.From[i] {
 			return false
 		}
 	}
 	return true
 }
+
+func (l *ConsolidatedDestRule) GetRule() *config.Config {
```

Comment:
Why do we need both this method and expose `Rule`

Reply:
The method is to avoid noisy nil check(`*ConsolidatedDestRule` could be nil)

exposing `Rule` is for the [unit test](https://github.com/istio/istio/pull/39730/files#diff-e7513f551cdd7e15ba1c7a36919002142c3a2ffac9116cab62af00d68a93bcb0R140), we can use a set func though